specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2019-09-01-preview/QueryPackQueries_API.json

```diff
@@ -330,7 +450,7 @@
     "LogAnalyticsQueryPackQueryProperties": {
       "description": "Properties that define an Log Analytics QueryPack-Query resource.",
       "properties": {
-        "queryId": {
+        "id": {
```

Comment:
This would be a breaking change.

Reply:
QueryPacks is a new resource under "microsoft.insights" RP and it is not yet exposed in any production region. no customer created any QueryPack yet, so this change cannot break any customer.